jobsAdmin/init.go

```diff
@@ -502,11 +502,12 @@ func GetJobSummary(jobID common.JobID) common.ListJobSummaryResponse {
 		// Job is completed if Job order is complete AND ALL transfers are completed/failed
 		// FIX: active or inactive state, then job order is said to be completed if final part of job has been ordered.
 		if (js.CompleteJobOrdered) && (part0PlanStatus.IsJobDone()) {
+			jm.ProcessMsg()
```

Comment:
For subtle things like this it helps to have a big comment to help the reader.

Plan file (hence part0PlanStatus.IsJobDone) is updated inline by jm.reportJobPartDoneHandler() while the various transfer stats are updated via xferDone messages processed by handleStatusUpdateMessage().
Now that the job has completed, make sure we drain all queued-but-not-processed-yet xferDone messages so as to return correct updated job status to our caller who may not call us again since we declare job completion.